src/Chapter1.hs

```diff
@@ -72,6 +72,7 @@ the `.hs` extension.
 -}
 module Chapter1 where
 
+import Data.Char
```

Comment:
Nice work on using imports! We explain them only in the next chapter, but it's good that you're already one step ahead 🦶🏻